bundles/ui/org.eclipse.smarthome.ui/src/main/java/org/eclipse/smarthome/ui/internal/chart/ChartServlet.java

```diff
@@ -135,6 +132,16 @@ protected void deactivate() {
         httpService.unregister(SERVLET_NAME);
     }
 
+    protected void modified(Map<String, Object> config) {
```

Comment:
Which means that the icon servlet code is wrong as it doesn't declare the modified method?